WMF Framework/WMFArticle+Extensions.swift

```diff
@@ -1,16 +1,6 @@
 import Foundation
 
 extension WMFArticle {
-    @objc public var displayTitleHTML: String {
```

Comment:
Had to move this to obj-c to use `clang diagnostic ignored "-Wdeprecated-declarations"` - looks like there's no equivalent in Swift yet